client/src/components/BizCard/BizCard.jsx

```diff
@@ -187,6 +192,13 @@ export default function BizCard({ props }) {
                   />
                   {Number(biz.price).toLocaleString()}
                 </div>
+                <hr className="divider" />
+                <div id="bizcard-user-review">
+                  <h2 id="reviews-header">Reviews</h2>
+                  {userReviews.map((review) => {
```

Comment:
just a heads up that you can just wrap the react element in parenthesis, and then you don't need a `return` keyword!